src/components/BookInput.js

```diff
@@ -1,28 +1,32 @@
 import { React, useState } from 'react';
-import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { postBooks } from '../redux/books/books';
 
-const BookInput = (props) => {
-  const [input, setInput] = useState({
-    title: '',
-    author: '',
-  });
+const categories = ['Choose a Category', 'Action', 'Action and Adventure', 'Drama', 'Mystery', 'Children Books', 'Romance', 'Comedy', 'Finances', 'Accounting', 'Novel', 'Horror', 'Science', 'Health', 'History', 'Business'];
+
+const BookInput = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState(categories[0]);
 
   const onChange = (e) => {
```

Comment:
Kindly rename this method for its purpose to be obvious. You can call it `onBookTitleChange`, for example.